bot/seasons/halloween/halloween_facts.py

```diff
@@ -30,37 +32,25 @@ def __init__(self, bot):
         with open(Path("bot", "resources", "halloween", "halloween_facts.json"), "r") as file:
             self.halloween_facts = json.load(file)
         self.channel = None
-        self.last_fact = None
+        self.facts = list(enumerate(self.halloween_facts))
+        random.shuffle(self.facts)
 
     async def on_ready(self):
-        self.channel = self.bot.get_channel(HACKTOBER_CHANNEL_ID)
+        self.channel = self.bot.get_channel(Hacktoberfest.channel_id)
         self.bot.loop.create_task(self._fact_publisher_task())
 
-    async def _fact_publisher_task(self):
-        """
-        A background task that runs forever, sending Halloween facts at random to the Discord channel with id equal to
-        HACKTOBER_CHANNEL_ID every INTERVAL seconds.
-        """
-        facts = list(enumerate(self.halloween_facts))
-        while True:
-            # Avoid choosing each fact at random to reduce chances of facts being reposted soon.
-            random.shuffle(facts)
-            for index, fact in facts:
-                embed = self._build_embed(index, fact)
-                await self.channel.send("Your regular serving of random Halloween facts", embed=embed)
-                self.last_fact = (index, fact)
-                await asyncio.sleep(INTERVAL)
+    @property
+    def random_fact(self):
+        return random.choice(self.facts)
 
-    @commands.command(name="hallofact", aliases=["hallofacts"], brief="Get the most recent Halloween fact")
-    async def get_last_fact(self, ctx):
+    @commands.command(name="spookyfact", aliases=["halloweenfact"], brief="Get the most recent Halloween fact")
```

Comment:
aliases should be a tuple

Reply:
Same as https://github.com/python-discord/seasonalbot/pull/64#discussion_r236063406

Reply:
The suggested change isn't a tuple though, you're missing a comma, it should be
```suggestion
    @commands.command(name="spookyfact", aliases=("halloweenfact",), brief="Get the most recent Halloween fact")
```

Reply:
Oh, Scragly already referenced it